docs/topics/mtr-rn-new-features-1_0_2.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-1_{context}"]
+= New features
+
+This section describes the new features of the {ProductName} ({ProductShortName}) 1.0.2.
+
+.New Rulesets
+{ProductShortName} includes new rulesets to support users migrating to EAP 8 and Hibernate 6.
```

Comment:
@HagayVider1 
I would suggest:
"...new rulesets that support migration to EAP 8 and Hibernate 6."

Reply:
Changed.